SlingingPlatformGame/Assets/Scripts/Slingshot.cs

```diff
@@ -76,46 +75,46 @@ public void CreatePlatformFromIndex()
     }
 
     public void StopPlatform(string PlatformName){
-        if (!StoppedPlatforms.Contains(PlatformName))
+        if (remainingPlatforms.Contains(PlatformName))
         {
-            StoppedPlatforms.Add(PlatformName);
+            remainingPlatforms.Remove(PlatformName);
         }
     }
 
 
     void CreatePlatform()
     {
+        if (!remainingPlatforms.Contains(selectedPlatform))
+        {
+            if (remainingPlatforms.Count != 0) {
+                selectedPlatform = (string)remainingPlatforms[0];

```

Comment:
won't this cause a problem when the user used up all the default platforms and still has ice ones left?

Reply:
yeah , so the change is - when default is all over, automatically ice is selected and vice versa